README.md

```diff
@@ -114,6 +114,24 @@ source and remove the line that says "#define USE_OPENCL".
     # Download and extract <https://sjeng.org/zero/best_v1.txt.zip> to msvc/x64/Release
     # msvc/x64/Release/leela-zero --weights weights.txt
 
+## Example of compiling and running - CMake (macOS/Ubuntu)
+
+    # Clone github repo
+    git clone https://github.com/gcp/leela-zero
+    cd leela-zero
+    # Use stand alone directory to keep source dir clean
+    mkdir build && cd build
+    cmake ..
+    make leelaz
+    curl -O https://sjeng.org/zero/best_v1.txt.zip
+    unzip best_v1.txt.zip
+    ./leelaz --weights weights.txt
+
+    # If you want to run unit tests too
+    git submodule update --init --recursive
```

Comment:
These instructions don't work. I think you now *require* the gtest stuff to be there before you can even run cmake.

Reply:
you are right, its now required to generate makefile, thanks for the correction